test/image/README.md

```diff
@@ -1,24 +1,46 @@
 # plotly.js image testing
 
-Test plotly.js with the Plotly Image-Server docker container.
+Test plotly.js with Plotly's image testing docker container.
 
+Requirements:
+- `docker` | [installation guidelines](http://docs.docker.com/engine/installation/)
+- `docker-machine` (for Mac and Windows users only) | [installation guidelines](https://docs.docker.com/machine/install-machine/)
+- `docker-compose` | [installation guidelines](https://docs.docker.com/compose/install/)
+
+### Step 0: Start the docker machine (Mac and Windows users only)
+
+Boot up docker machine (named `default`):
+
+```bash
+docker-machine start default
+```
+
+Set up the docker environment for `docker-compose`:
+
+```bash
+eval $(docker-machine env default)
```

Comment:
A good shell function to add to your rc file if you have lots of docker-machines (I have like 8 that I use depending) is [this](https://github.com/mdtusz/dotfiles/blob/master/.shellfn#L32).